svelte.config.js

```diff
@@ -1,16 +1,24 @@
-import { vitePreprocess } from '@sveltejs/kit/vite';
-import adapter from '@sveltejs/adapter-auto';
+import { vitePreprocess } from '@sveltejs/vite-plugin-svelte';
+import adapter from '@sveltejs/adapter-static';
 
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
+	// Consult https://kit.svelte.dev/docs/integrations#preprocessors
+	// for more information about preprocessors
+	preprocess: [vitePreprocess({})],
 	kit: {
 		// adapter-auto only supports some environments, see https://kit.svelte.dev/docs/adapter-auto for a list.
 		// If your environment is not supported or you settled on a specific environment, switch out the adapter.
 		// See https://kit.svelte.dev/docs/adapters for more information about adapters.
-		adapter: adapter()
+		adapter: adapter({
+			pages: 'dist',
+			assets: 'dist',
+			strict: false,
+		})
 	},
-
-	preprocess: [vitePreprocess({})]
+	vitePlugin: {
+		inspector: true
```

Comment:
same with svelte config. IDK what any of this means but it LGTM!

Reply:
haha can assure you every line is needed! except maybe the inspector (open the application and press ctrl-shift (or something else if you're on apple? command-shift maybe?))